locale/en/submission.po

```diff
@@ -2416,12 +2416,15 @@ msgstr "Active submissions"
 msgid "submission.dashboard.view.needsEditor"
 msgstr "Needs editor"
 
-msgid "submission.dashboard.view.initialReview"
-msgstr "All in desk/initial review"
+msgid "submission.dashboard.view.submissionStageAll"
+msgstr "All in submission stage"
```

Comment:
Not sure if we should change the wording here and other places

Reply:
The [ticket](https://github.com/pkp/pkp-lib/issues/10835#issue-2804871826) does mention that the labels should be updated to match this [design](https://www.figma.com/design/Wf7sDlUg2372jaKKTJ0Mgz/OJS-3.4-3.5?node-id=9303-11153&t=UByqbCDCSnXlfqpd-4), but @jardakotesovec or @Devika008 can confirm if this is still the case.

Reply:
Yeah, this is correct change. 'Desk review' was supposed to replace 'submission' stage in early designs..  But we back out from that.